cli/app.go

```diff
@@ -36,7 +40,9 @@ If run from a git repository with a 'flynn' remote for the app, it will be
 removed.
 
 Options:
-	-y, --yes  Skip the confirmation prompt.
+	-r, --remote <remote>  Name of git remote to delete, empty string for none. [default: flynn]
+	-y, --yes              Skip the confirmation prompt.
+	--app <name>           Explicitly set name of app to delete (otherwise found automatically if --remote is not an empty string).
```

Comment:
There is already a global [`-a` flag](https://github.com/flynn/flynn/blob/bd2ac6ed88ad4ef4192da84ecde1545e8a5fab0b/cli/main.go#L93-L103), so `--app` should be redundant.